web/src/pages/tasks/[taskUrlSlug].tsx

```diff
@@ -202,6 +202,7 @@ export const TaskElement = (props: { task: Task; children?: ReactNode | ReactNod
       <div>
         <TaskHeader task={props.task} />
         {props.children}
+        <Content>{props.task.summaryDescriptionMd || ""}</Content>
```

Comment:
This change is necessary because otherwise the summary did not show up if the task contained location-dependent content or post-onboarding content.

This led to the Town Mercantile License task not having any content displayed to the user, because it has a location-dependent section